runtime/kusama/src/lib.rs

```diff
@@ -1491,6 +1491,8 @@ pub type Migrations = (
 	// Unreleased - add new migrations here:
 	pallet_nomination_pools::migration::v5::MigrateToV5<Runtime>,
 	parachains_configuration::migration::v5::MigrateToV5<Runtime>,
+	/* Asynchronous backing mirgration */
```

Comment:
nit: I think we should prefer consistent comment syntax?
```suggestion
	// Asynchronous backing mirgration
```